### PR TITLE
Jr/integ 383 ensure all credentials pass with test poetry deps install

### PR DESCRIPTION
**Pull request by johnathan-rodriguez**

### CODE REVIEW

File: shipyard_blueprints/azureblob/pyproject.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ shipyard-templates = "0.5.0a0"
 shipyard-utils = "^0.1.4"
 
 [tool.poetry.group.dev.dependencies]
-shipyard-templates = {path = "../../shipyard-templates", develop = true}
+#shipyard-templates = {path = "../../shipyard-templates", develop = true}
 pytest = "^7.4.0"
```
Comment (wrp801) [inline shipyard_blueprints/azureblob/pyproject.toml:18]: Not sure if this older version of pytest will break anything

**Review by wrp801 — APPROVED** [review]

**Comment by ericelsken** [comment]
I pulled and ran, but I'm getting 
```
Running 'poetry install' for ./shipyard_blueprints/airbyte/ ...
Installing dependencies from lock file

pyproject.toml changed significantly since poetry.lock was last generated. Run `poetry lock [--no-update]` to fix the lock file.
```

Is there an update to the `poetry install` command that we need to avoid this error in the future? Or does more code need to be committed so this doesn't happen? 

Ie. do we intend to fail on this error in the future and we fix it now, or do we not care about this error and update the script to avoid it?

**Comment by wrp801** [comment]
This is being fixed, we need to add the updated lock files to the repo

**Comment by ericelsken** [comment]
The script worked with a warning. Do y'all need to address? 
```
Running 'poetry install' for ./shipyard_blueprints/smartsheets/ ...
Creating virtualenv shipayrd-smartsheets--FqSXwfM-py3.9 in /root/.cache/pypoetry/virtualenvs
Installing dependencies from lock file

Package operations: 24 installs, 0 updates, 0 removals

  - Installing certifi (2024.2.2)
  - Installing charset-normalizer (3.3.2)
  - Installing idna (3.6)
  - Installing urllib3 (2.2.1)
  - Installing requests (2.31.0)
  - Installing six (1.16.0)
  - Installing click (8.1.7)
  - Installing et-xmlfile (1.1.0)
  - Installing mypy-extensions (1.0.0)
  - Installing numpy (1.26.4)
  - Installing packaging (23.2)
  - Installing pathspec (0.12.1)
  - Installing platformdirs (4.2.0)
  - Installing python-dateutil (2.8.2)
  - Installing pytz (2024.1)
  - Installing requests-toolbelt (1.0.0)
  - Installing tomli (2.0.1)
  - Installing typing-extensions (4.10.0)
  - Installing black (23.12.1)
  - Installing openpyxl (3.1.2)
  - Installing pandas (1.5.3)
  - Installing python-dotenv (1.0.1)
  - Installing shipyard-templates (0.4.1)
  - Installing smartsheet-python-sdk (3.0.2)

Installing the current project: shipayrd-smartsheets (0.1.0)
Warning: The current project could not be installed: No file/folder found for package shipayrd-smartsheets
If you do not want to install the current project use --no-root.
If you want to use Poetry only for dependency management but not for packaging, you can set the operating mode to "non-package" in your pyproject.toml file.
In a future version of Poetry this warning will become an error!
```

**Comment by wrp801** [comment]
Yes I got the same error - let me take a look at this real quick. This should be quick fix

**Pull request merged**